From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: aswin-mcw <aswin.kalies@multicorewareinc.com>
Date: Wed, 9 Sep 2026 13:14:24 +0530
Subject: [PATCH 046/109] Cygwin: hookapi.cc: Fix some handles not being
 inherited when spawning

(cherry picked from commit fdc9a747fe22dd604971235a2792e130650dd74e)
---
 winsup/cygwin/hookapi.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/winsup/cygwin/hookapi.cc b/winsup/cygwin/hookapi.cc
index 9f31a716c..169364bce 100644
--- a/winsup/cygwin/hookapi.cc
+++ b/winsup/cygwin/hookapi.cc
@@ -45,6 +45,8 @@ PEHeaderFromHModule (HMODULE hModule)
     {
     case IMAGE_FILE_MACHINE_AMD64:
       break;
+    case IMAGE_FILE_MACHINE_ARM64:
+      break;
     default:
       return NULL;
     }
